Scripts/Enemies/FirstBoss/first_boss.gd

```diff
@@ -10,7 +10,7 @@ var changeScale : bool = false
 @onready var animation : AnimatedSprite2D = $FirstBossSprites
 @onready var damageParticles : GPUParticles2D = $FirstBossSprites/Node2D/DamageParticle
 
-var speed : float = 35.0
+var speed : float = 70.0
```

Comment:
I still think this is way too fast for a normal difficulty, especially for the first enemy you meet and if you're new to playing it, ha.